rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1234,6 +1239,17 @@ private void processPegoutsInBatch(
         }
     }
 
+    private void savePegoutTxSigHash(BtcTransaction pegoutTx) {
+        if (!activations.isActive(ConsensusRule.RSKIP379)){
```

Comment:
Should this check be outside of this method surrounding the call to this method so it is clear when we want to execute this method? I think we have talked about that convention.

Reply:
No, in this case, it's fine to have the check inside the method since it is private and only used internally.